rasa/data.py

```diff
@@ -167,6 +168,34 @@ def is_story_file(file_path: Text) -> bool:
         return False
 
 
+def is_conversation_test_file(file_path: Text) -> bool:
+    """Checks if a file is a Rasa conversation test file.
+
+    Args:
+        file_path: Path of the file which should be checked.
+
+    Returns:
+        `True` if it's a conversation test file, otherwise `False`.
+    """
+
+    if not file_path.endswith(".md"):
+        return False
+
+    try:
+        dirname = os.path.dirname(file_path)
+        return is_story_file(file_path) and DEFAULT_E2E_TESTS_PATH in dirname
```

Comment:
optional: I think that's a bit quicker since the `is_story_file` test is a bit slower due to trying to read the file.
```suggestion
        return DEFAULT_E2E_TESTS_PATH in dirname and is_story_file(file_path)
```